models/ed/inst/template.job

```diff
@@ -45,7 +45,7 @@ if [ ! -e "@OUTDIR@/history.xml" ]; then
 
   # convert to MsTMIP
   echo "require (PEcAn.ED2)
-model2netcdf.ED2('@OUTDIR@', @SITE_LAT@, @SITE_LON@, '@START_DATE@', '@END_DATE@')
+model2netcdf.ED2('@OUTDIR@', @SITE_LAT@, @SITE_LON@, '@START_DATE@', '@END_DATE@', '@PFTNAMES@', '@DHBREAKS@')
```

Comment:
Do we really want model2netcdf.ED2 to only process specific PFTs and DBH bins? I'd expect the default to be all PFTs and size classes

Reply:
it processes all, these arguments have multiple elements e.g.

```
pft.names = c("temperate.Early_Hardwood", "temperate.Late_Conifer", "temperate.Late_Hardwood")
dbh.brakes = c(0,50,100)
```
but I think pft.names should be possible to extract from the output file itself on the fly

Reply:
Yes, I understood that these variables can be vectors, but the code doesn't seem to allow them to have a default. Because it's not a 'standard' pecan setting, if someone doesn't manually add DHBREAKS to their pecan.xml via Advanced Edit, it appears that job.sh will throw an error.